src/qforte/utils/circuit_to_latex.py

```diff
@@ -0,0 +1,145 @@
+import qforte
+
+def circuit_to_latex(circ):
+    """
+    This function constructs a latex file with
+    the graphical representation of the given
+    quantum circuit.
+
+    Depending on how large the circuit is,
+    compiling with lualatex is advisable.
+
+    The quantikz package is required for
+    compiling the tex file.
+
+    Arguments
+    =========
+
+    circ: Circuit object
+        The quantum circuit that we want to plot.
+
+    Returns
+    =======
+
+    circ.tex: latex file
+        The latex file containing the plot of the
+        given quantum circuit.
+    """
+
+    preamble = r"""\documentclass[tikz]{standalone}
+
+\usepackage{quantikz}
+
+\begin{document}
+"""
+
+    tikz_start = r"""
+\begin{tikzpicture}
+    \node {
+        \begin{quantikz}
+"""
+
+    tikz_end = r"""        \end{quantikz}
+    };
+\end{tikzpicture}
+"""
+
+    epilogue = r"""
+\end{document}"""
+
+    texfile = open("circuit.tex", "w")
```

Comment:
Should the filename be an optional parameter?

Reply:
Good point!